x/tokenfactory/types/msgs.go

```diff
@@ -74,7 +74,7 @@ func NewMsgMintTo(sender string, amount sdk.Coin, mintToAddress string) *MsgMint
 
 func (m MsgMint) Route() string { return RouterKey }
 func (m MsgMint) Type() string  { return TypeMsgMint }
-func (m MsgMint) ValidateBasic() error {
+func (m MsgMint) Validate() error {
```

Comment:
no, we cannot just to that because [if the address is empty the sender address is used](https://github.com/neutron-org/neutron/blob/1c2388b321f215a024923aefc49cb7a929a8dfad/x/tokenfactory/keeper/msg_server.go#L108-L110). we should do the check only if the sender addr is not an empty string

Reply:
Done